src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import axios from 'axios';
 
 import getFeatureLoggerInstance, {EFeatureLogFlag} from '../../../utils/log/FeatureLog';
+import {GetstreamWebsocket} from './types.d';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `GetstreamWebsocket` is added but it's not clear where this type is defined. Please ensure that the type is correctly defined and exported in the `./types.d` file.
```diff
+ import {GetstreamWebsocket} from './types.d';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->